src/VarUnit.h

```diff
@@ -23,6 +24,7 @@
 
 
 #include "pre_guard.h"
+#include <QApplication>
```

Comment:
I'm not entirely sure why this include is added?

Reply:
For the `Q_DECLARE_TR_FUNCTIONS(` ... classname ... `)` macro which is needed to get the `tr(`...`)` method/function to putt strings through the Qt translation system if a class is NOT derived from the `QObject` class (in ***those** cases you'd use the `QOBJECT` macro) - this is a (lighter, I guess) macro that only adds in stuff for the translation system without all the other things (Signal/Slots, metaproperties) that QOBJECT provides.

Reply:
Good enough for me, thanks for explaining =)